libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -330,6 +337,9 @@ export async function fetchAndProcessPlainHtml({ url, shouldDecorateLinks = true
     path = mepFragment.target;
   }
   const res = await fetch(path.replace(/(\.html$|$)/, '.plain.html'));
+  if (res.status !== 200) {
+    throw new Error(`${res.statusText} url: ${res.url}`);
```

Comment:
Again, not sure if throwing errors is the right idea, especially if the PR resolves around lana logs as the title indicates

Reply:
This code is part of fetchAndProcessPlainHtml. Although we already have LANA logs for this, they don't work if the HTML assets fail to load. 
Refer to these code sections:
[Code Snippet 1](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/global-navigation.js#L1008-L1014)
[Code Snippet 2](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-footer/global-footer.js#L72-L82)

To make it work with the existing logs, an error throw has been added. Previously, if there was no content or if HTML assets failed to load, the code execution would also stop, as shown in Code Snippet 1 [here](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/global-navigation.js#L1014) and similarly in Code Snippet 2.

Reply:
Same here as well.
To log it near to the failure will be a good approach and easy for debugging the error as well.